client/dashboard/app/router/domains.ts

```diff
@@ -548,7 +548,7 @@ export const domainTransferIndexRoute = createRoute( {
 		}
 
 		if ( domain.transfer_status === DomainTransferStatus.PENDING_START ) {
-			if ( domain.last_transfer_error === null ) {
+			if ( ! domain.last_transfer_error ) {
```

Comment:
The `last_transfer_error` can be an empty string, so checking if it's a falsy value works here.